appveyor.yml

```diff
@@ -79,10 +63,6 @@ install:
     - cmd: conda install -n root --quiet --yes conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
 
-    # conda-build 2.0.12 may have broken build?
-    - cmd: conda install --yes --quiet conda-build=2.0.11
```

Comment:
@ccordoba12 [suggested](https://github.com/conda-forge/cairo-feedstock/pull/16#issuecomment-268428969) that maybe conda-build 2.0.12 was behind the weird new Windows build failures. I don't think that seems to be the case so this change should be dropped.